pkg/syncer/metadatasyncer.go

```diff
@@ -1138,7 +1138,13 @@ func csiUpdatePod(ctx context.Context, pod *v1.Pod, metadataSyncer *metadataSync
 					continue
 				}
 			} else {
-				log.Warnf("CSI migration feature state is disabled")
+				// For vSphere volumes we need to log the message that CSI migration feature state is disabled
+				if volume.VsphereVolume != nil {
+					log.Debugf("CSI migration feature state is disabled")
```

Comment:
The log message is a static string. How is this log message even helping?

Reply:
This will help while debugging. And also we do not want to flood the logs for system pods since many of the system pods do not have PVC